recipes/ubermag/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ubermag" %}
+{% set version = "0.1" %}
+{% set sha256 = "8cacbb83f1f3c61338194ae83029c06b54ec9f4a10cb9ec365cccac16b42f0d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps . -vvv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - oommfc
+    - pytest
```

Comment:
do you really need ``pytest`` to run this project or just to test it?
if it is just to test you can add ``pytest`` on the section ``test``, ``requires``
```suggestion
```